README.md

```diff
@@ -44,8 +46,8 @@ key.alias=key-alias
 key.password=key-password
 ```
 
-Run `./gradlew assembleRelease` to build the package. The package can be installed using Android package manager.
+Run `./gradlew assembleRelease` to build the package, which can be installed using the Android package manager.
 
 ## License
 
-Foxy Droid is available under the terms of GNU General Public License v3 or later. Copyright (C) 2020 kitsunyan.
+GPLv3+ copylefted libre software. Copyright © 2020 kitsunyan.
```

Comment:
(wtf, where my original comment has gone)

I'd preferred to leave it as it is. I use the same line for my other project. It seems that it's also better to use (C) instead of ©, especially since it's the only place where copyright information is mentioned (I didn't add headers into the code).

Reply:
A link to the source code would do nicely here. (C) doesn't actually mean anything, from what I can remember. Headers? © is just UTF-8.
Arguably "Copyright © 2020 kitsunyan." isn't strictly needed either (I think), but having it isn't wrong.

Reply:
> isn't strictly needed either

Actually, yes. I don't mind removing it. But I want to keep "Foxy Droid is available under the terms of GNU General Public License v3 or later."

Reply:
> (C) doesn't actually mean anything, from what I can remember. 

I could confuse them. This is why I want to remove "Copyright" information.

> Headers? © is just UTF-8.

I mean, it's common to place a header into each file of the project with legal information. I don't want to do this.

Reply:
> Foxy Droid

already know what the program is.

>is available under the terms of

this sounds like the availability is limited, and it sounds like the terms are going to be limiting its availability, when one could argue the opposite is true.

> GNU General Public License v3 or later.

there should be a "the" here, and a comma. v3 matches GPL. "version" matches "GNU General Public License", similar to + vs "later".
The only piece of valuable information is that GPLv3 is a license.
"GPLv3+ licensed, copylefted libre software" would suffice.

Reply:
> already know what the program is.

Yes, think about this line like about something said by a lawyer :) (I'm not a lawyer ofc). So it's okay to leave it here.

> this sounds like the availability is limited

Yes, it's my right since I'm a copyright holder :> So I'm making it available only under GPL v3 or any later version. I don't make it available under any EULA, Apache License, MIT License, or anything else. Only GPL v3 or any later! As I said, this like is more "strict", especially since it's the only place I'm mentioning copyright information across the whole project repository.

> there should be a "the" here

Probably, yes. Even on gnu.org they use "the". Can be changed.

> and a comma

Comma is definitely not necessary (since there are only 2 options, "v3 or later"; comma is used when there are 3 and more options, like "v2, v3, or later", just an example).

Reply:
What I meant is that the availability is ensured with GPLv3+, and it should ideally sound like it for people that don't know. Oxford comma makes the second option immediately distinct, though not needed here, agreed.
"Copyleft" is a copyright concept. It is already clear this is your copyrighted material from the name being next to "copyright", and the symbol for it. Spending a sentence on what the libre software+ copyleft freedoms entail is better spent. "Use, see, change and share at will, with all." or some-such works. I am not sure about the intention here, but legal matters are maintained. The important part is presenting the project.

Reply:
> availability is ensured with GPLv3+

That's not quite right. License it something like a "contract": I'm giving a copy of the program under some conditions. So when I say "available under GPL", that means "by using this program you're agreeing with the terms of GPL v3 or any later". I don't provide the choice of another "contracts", only listed one. What you're saying about "availability" is ensured by copyleft _after_ user gets a copy.

But what is really ensured by GPL is that the software is "libre" and "copylefted". In my opinion, your "copylefted libre software" feels kinda "buzzwordy" to me (I noticed you like to place "copylefted libre software" to other projects) (not like I'm trying to shame you or something like that), which is diametrically opposed to my vision of this line to be "strict". There are other words which can describe the program, not only "libre" and "copylefted": "free from patent prosecution", "tivoization-prohibited", etc, etc. But for some reason you want to mention only "libre" and "copylefted", which makes it even more "buzzwordy".

Ultimately, it's about personal preference: make this line "interesting and joyful" or "boring and strict". I explained why I want the second option. I also think, when it comes to something about "legal", it's more important to explain restrictions, than freedoms, so no one could be fooled.

Although, it's a good idea to insert "the" and replace copyright symbol.

Reply:
From the point of view of the user, copyleft is a premise of availability. I agree my explanation didn't take into account the specifics of how. "libre" and "copyleft" are the only points made, and those are the only ones qualifying it over not being so. Libre is chosen because it is unencumbered with the FSF and the OSI, and it is copyleft as an addition to that. It carries nothing but qualifying statements, and it seems to work a lot better than FSF language. "Full" and "strong" copyleft is more specifically what it is. The patent litigation point is a good one to make.
It can be welcoming and strict. Boring and unwelcoming is what I want to get away from. Made a point to explain it in a way that makes a point of the copyleft requirement.